docs/APIs/tasks/rtos.md

```diff
@@ -132,16 +132,16 @@ This example shows ``Queue`` and ``MemoryPool`` (see below) managing measurement
 
 <span class="images">![](Images/Thread/mail_queue.png)</span>
 
+### Mail class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.3/api/classrtos_1_1Mail.html)
+
 ### Mail example
 
 This code uses ``mail`` to manage measurement.
 
 [![View code](https://www.mbed.com/embed/?url=https://developer.mbed.org/teams/mbed/code/rtos_mail/)](https://developer.mbed.org/teams/mbed/code/rtos_mail/file/tip/main.cpp)
 
-### Mail class reference
-
-[![View code](https://www.mbed.com/embed/?type=library)](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.3/api/classrtos_1_1Mail.html)
-
 ## RtosTimer
 
 <span class="warnings">**Warning**: DEPRECATED</br>
```

Comment:
@iriark01 Please note that the RtosTimer goes away entirely in a later pull request, so you can ignore the "DEPRECATED" warning as long as we merge the other pull request.